utils/oscap-im

```diff
@@ -111,9 +111,22 @@ def pre_scan_fix(args):
             "--output", remediation_script.name]
         add_common_args(args, gen_fix_cmd)
         gen_fix_cmd.append(args.data_stream)
-        subprocess.run(gen_fix_cmd, check=True)
-        subprocess.run(["bash", remediation_script.name], check=True)
-
+    try:
+        subprocess.run(gen_fix_cmd, check=True, capture_output=True)
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(
+            f"OpenSCAP generate fix failed with return code {e.returncode}.\n"
+            f"Output: {e.stderr.decode()}")
+    try:
+        subprocess.run(
```

Comment:
The remediation script will install packages and sometimes it might be useful to see the whole dnf output. So here I would remove `capture_output=True` so we see both stdout and stderr combined together as originally outputted by dnf and in `RuntimeError` exception we can just print the failed with return code message (current first line).